File: RB101_Programming_Foundations/small_problems/Easy9/welcome_stranger.rb

def greetings(array, hash)
  full_name = array.join(' ')
  occupation = hash.values.join(' ')

  "Hello, #{full_name}! Nice to have a #{occupation} around."
end




puts greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' })

File: js210_fundamentals_of_JS_Programming/objects/welcome_stranger.js
function greetings(nameParts, jobParts) {
  let nameGreet = `Hello, ${nameParts.join(' ')}!`;
  let jobGreet = `Nice to have a ${jobParts.title} ${jobParts.occupation} around.`;
  console.log(nameGreet + jobGreet);
}
greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' });

// console output
// Hello, John Q Doe! Nice to have a Master Plumber around.
